qokit/labs.py

```diff
@@ -579,6 +601,4 @@ def get_gate_optimized_terms_greedy(N: int, number_of_gate_zones: int = 4, seed:
                 j += 1
             k += swapped
 
-    assert set(circuit) == set(get_terms(N))
```

Comment:
Any reason to drop this check? Now that `get_terms` returns weights well, should this become `assert set(circuit) == set(get_term_indices(N))` or something like that?

Reply:
This is already checked with [tests](https://github.com/jpmorganchase/QOKit/blob/46b1f517a3c7014989dd5f724c056b9ff79877f9/tests/test_labs.py#L51-L52), so runinng `get_terms` seemed redundant. We can keep the assert but then the tests are a bit silly:)

Reply:
Ah, right! Thanks for catching